src/index.md

```diff
@@ -4,14 +4,78 @@
+
+Once you have the required changes, a Committer in the project can start a vote by sending an email proposing that you should become a Committer. The email should follow the template found in the [OpenJDK Project description](https://openjdk.java.net/projects/).
+
+A Committer is allowed to push changes without the aid of a sponsor. A Committer is also allowed to nominate other non-Committers to become Committers in the project.
+
+### Becoming a Reviewer
+
+To become a [Reviewer](https://openjdk.java.net/bylaws#reviewer) you must show a track record of sound and correct judgement calls as mentioned above. Being a good Committer doesn't necessarily make you a good Reviewer. As a Reviewer you have the power to approve changes for inclusion into the project source code. This means that a Reviewer needs to be able to judge the quality and appropriateness of any proposed change, not just the mechanics of the code.
+
+The assumption is that after having produced 32 significant changes one should have become familiar with the process around reviews and the requirements around getting a change approved. This should really be seen as a minimum requirement though. A more practical consideration would be to look at whether the non-trivial commits of a potential Reviewer are accepted largely intact or whether they are always being refined by the review process. There may be cases where it will take significantly more than 32 changes for a Committer to be ready to become a Reviewer.
+
+Once you are deemed ready, a Reviewer in the project can start a vote by sending an email proposing that you should become a Reviewer. The email should follow the template found in the [OpenJDK Project description](https://openjdk.java.net/projects/).
+
+### Non-trivial/Significant changes
+
+One key definition when advancing through the OpenJDK roles is the significant change. What exactly does it take for a change to be significant?
+
+Instead of describing the significant change (because that's quite difficult to define) provided here is a few examples of changes that wouldn't be considered significant or for other reasons wouldn't count as significant contributions.
+
+* Purely aesthetic changes like renaming or fixing indentation
+* Repeated follow-up bugfixes from earlier changes
+* Larger changes where only a non-significant portion of the work was done by the Contributor under vote
+* Trivial backports of someone else's changes
```

Comment:
The word "significant" is used in the description of the 32 changes required for the Reviewer role as well. I agree that the significance of the commits are of less importance in the Reviewer case and I intended for the text to show that there are other criteria that are more important.
I would prefer to keep this section where it is. I see it as kind of a foot-note and I think it would take too much space if placed in the Committer section. I assume that you noted that it's not in the Reviewer section, it's below the Reviewer section. That may not be obvious when looking at the markdown.